freeformseries1/createfromonprem/createfromonprem.md

```diff
@@ -0,0 +1,62 @@
+# Create FreeForm application using an Oracle Essbase Outline
+
+## Introduction
+
+One of the main use cases for FreeForm application is the ease with which an Essbase outline can be used to build FreeForm application in EPM Cloud. 
+	![Image alt text](images/usecasesummary.png)
+In this lab, we will use an existing Essbase outline for Aggregate Storage Option (ASO) and create a FreeForm application from it.
+
+Estimated Lab Time: 7 minutes
+
+### About
+Use EPM Cloud FreeForm application to migrate on-premises Essbase outline. The process will use an outline file from on-premises application to build the FreeForm ASO application.
+
+### Objectives
+
+In this lab, you will:
+* Import an ASO outline to EPM Cloud
+* Create a FreeForm application using the ASO outline
+
+### Prerequisites
+
+This lab assumes you have:
+* Access to an Essbase on-premises outline file
+* An Oracle EPM Enterprise Cloud subscription
+* Service administrator rights to the instance where you are going to create the FreeForm application
+
+
+## Task 1: Creation Process
+
+1. Hit **Start** under **Create a new application**.
+
+	![Image alt text](images/selectstartonprem.png)
+
+2. Populate **Name** and **Description** for the FreeForm application being created.
+
+  ![Image alt text](images/namedescription.png)
```

Comment:
Add alternate text to all the images. For example, in this case, you can add the alternate text as  Name Description for namesdescription.png